### ISSUE TITLE
Python lib files missing

### ISSUE THREAD

**Issue by Malifex** [ExposureSoftware/TEC-Client#58]
I'm not a pure programmer so I'm unsure what's causing this. After entering my credentials to log in, the client hangs at "signing in..." and has the following in the console:

10
Starting logger.
'Starting connection!'
Exception in thread Thread-1:
Traceback (most recent call last):
  File "c:\Program Files\Python 3.3\lib\site-packages\requests\packages\urllib3\
util\ssl_.py", line 267, in ssl_wrap_socket
FileNotFoundError: [Errno 2] No such file or directory

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Program Files\Python 3.3\lib\site-packages\requests\adapters.py", lin
e 370, in send
  File "c:\Program Files\Python 3.3\lib\site-packages\requests\packages\urllib3\
connectionpool.py", line 544, in urlopen
  File "c:\Program Files\Python 3.3\lib\site-packages\requests\packages\urllib3\
connectionpool.py", line 341, in _make_request
  File "c:\Program Files\Python 3.3\lib\site-packages\requests\packages\urllib3\
connectionpool.py", line 761, in _validate_conn
  File "c:\Program Files\Python 3.3\lib\site-packages\requests\packages\urllib3\
connection.py", line 238, in connect
  File "c:\Program Files\Python 3.3\lib\site-packages\requests\packages\urllib3\
util\ssl_.py", line 269, in ssl_wrap_socket
requests.packages.urllib3.exceptions.SSLError: [Errno 2] No such file or directo
ry

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\Python\64-bit\3.3\lib\threading.py", line 637, in _bootstrap_inner
  File "c:\Python\64-bit\3.3\lib\threading.py", line 594, in run
  File "d:\Marshall\Documents\Code\Python Code\TEC Client\client\client.py", lin
e 78, in listen
  File "d:\Marshall\Documents\Code\Python Code\TEC Client\client\client.py", lin
e 133, in login_user
  File "c:\Program Files\Python 3.3\lib\site-packages\requests\api.py", line 108
, in post
  File "c:\Program Files\Python 3.3\lib\site-packages\requests\api.py", line 50,
 in request
  File "c:\Program Files\Python 3.3\lib\site-packages\requests\sessions.py", lin
e 465, in request
  File "c:\Program Files\Python 3.3\lib\site-packages\requests\sessions.py", lin
e 573, in send
  File "c:\Program Files\Python 3.3\lib\site-packages\requests\adapters.py", lin
e 431, in send
requests.exceptions.SSLError: [Errno 2] No such file or directory

I'm wondering if I installed correctly. I even downloaded python 3.3 and installed to the above directory with no luck.

Let me know if I'm doing something wrong or if this is a bug. Thanks.

OS: Windows 7 x64

**Comment by marshall-davis** [comment]
Some questions for you here:
1. Using the installer to install the client?
2. If not do you have `pip` installed and are you installed dependencies by running `pip install -r requirements.txt` in the project directory?

**Comment by PatrickDattilio** [comment]
I got the same error when building a release. Adding `verify="false"` when we do our login seemed to fix it.

**Comment by marshall-davis** [comment]
Oh, yeah. It's trying to connect via HTTPS and the certificates for Skotos.net are goofy.

The current state of the `dev` branch has 

```
response = requests.post(url, headers=header, data=data, allow_redirects=False, verify=False)
```

This should resolve the issue.

**Comment by marshall-davis** [comment]
This should be fixed in the most recent release (v0.5.3-alpha)
